src/guides/v2.3/comp-mgr/bk-compman-upgrade-guide.md

```diff
@@ -53,6 +53,8 @@ Related topics
 
 Complete the tasks discussed in [Prerequisites].
 
+ {:.bs-callout-info}
+See the [Safe upgrade tool section]({{ site.baseurl }}/safe-upgrade-tool/introduction.html) for more information on the new CLI magento tool that helps you update your magento software.
```

Comment:
```suggestion
See the [Safe Upgrade Tool ]({{ site.baseurl }}/safe-upgrade-tool/introduction.html) for more information about the new Magento CLI tool that helps you update your Magento software.
```

Reply:
Could we add "(ALPHA)" next to Safe Upgrade Tool ?

Reply:
added